src/main/java/com/spartronics4915/frc2019/subsystems/Superstructure.java

```diff
@@ -274,7 +274,7 @@ else if (mWantedState == WantedState.ALIGN_AND_SHOOT_CARGO_BAY
                         if (mStateChanged)
                         {
                             mCargoChute.setWantedState(CargoChute.WantedState.LOWER);
-                            mPanelHandler.setWantedState(PanelHandler.WantedState.EJECT);
+                            mPanelHandler.setWantedState(PanelHandler.WantedState.ARM_DOWN);
```

Comment:
!! someone has to add more superstructure states to differentiate between ejecting cargo to the rocket and to the cargo ship. currently this is only set up to eject to the cargo ship